Jenkinsfile

```diff
@@ -148,7 +148,7 @@ pipeline {
                                            mgmtNetwork, testNetwork,
                                            env.TESTBED_TEMPLATE, env.CONTROLLER_TEMPLATE)
 
-                            provisionTestEnv(vmwareConfig)
+                            provisionTestEnv(vmwareConfig, inventoryFilePath)
```

Comment:
When `controller` role will be added to `vmware-deploy-testenv.yml`, we can remove the second parameter to `provisionTestEnv`